### PR TITLE
#955: Adjusted the dropdowns for the "Actions" button under Project V…

### LINKED ISSUE #955: Right align actions menu and line above delete
### Description

We want the actions menu on projects, workpackages, and change requests to be right aligned with the actions button. We also want a line that seperates the delete button with the other buttons. There should also be a trash icon for the delete like there are icons for the other options.

### Acceptance Criteria

^

### Proposed Solution

View the Change request delete for what the line should look like, but I believe it should be right aligned not left aligned as it is right now. So right align all of them and make sure they all have a line seperating delete from the other options

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by XHoke04**
## Right align actions menu and line above delete

Adjust the dropdown for the "Actions" button under Project Review, WorkPackage View, and ChangeRequest Action Menu, so that they are right aligned with the "Actions" box. Also added a divider above each "Delete" button in the "Actions" dropdown. Switched the order of two buttons in the actions dropdown under Project View. Switched "Unassign from My Team" and "Suggest Budget Increase"

## Screenshots
Screenshot 2024-01-25 at 10 49 46 AM

Screenshot 2024-01-25 at 10 50 34 AM

Screenshot 2024-01-25 at 10 51 25 AM

Screenshot 2024-01-25 at 10 51 59 AM

Screenshot 2024-01-25 at 10 52 23 AM

Screenshot 2024-01-25 at 10 52 48 AM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #955

### CODE REVIEW

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx [hunk 1]
```diff
@@ -176,8 +177,8 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
       >
         <EditButton />
         <CreateChangeRequestButton />
-        <SuggestBudgetIncreaseButton />
```
Comment (ryanhowe28) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx:179]: Is there are reason you switched these around?

**Review by ryanhowe28 — COMMENTED** [review]
Also dont need to include your name in the branch name. Just a small description of the ticket is good, after the ticket number

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx (hunk 1 above)
Comment (XHoke04) [reply src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx:179]: If you try to add divider={true} to the "Unassign from My Team" button, only team leads will be able to properly see the implemented changes.

Comment (ryanhowe28) [reply src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx:179]: Got it

**Review by ryanhowe28 — APPROVED** [review]
LGTM

File: src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx [hunk 2]
```diff
@@ -145,7 +146,19 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
       >
         Actions
       </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+      <Menu
```
Comment (RChandler234) [inline src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx:149]: can you update this one to use the ActionMenu component as well

**Review by RChandler234 — CHANGES_REQUESTED** [review]
the reimbursement request detail page, and main finance page are a bit off:
image
image

File: src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx (hunk 2 above)
Comment (XHoke04) [reply src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx:149]: Yup. Didn't know those two pages needed fixing mb. Do you want the delete button moved to the bottom with a divider added? (top image you sent)

Comment (RChandler234) [reply src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx:149]: yes that would be awesome

**Comment by RChandler234** [comment]
There's been no progress made on this in a while, message me on slack if you wanna keep working on it

**Pull request closed without merging**